README.rst

```diff
@@ -182,46 +182,33 @@ First:
   $ git commit -a --amend
   $ git push
 
-Then open a PR, and when the PR is approved:
+- Open a PR, and when the PR is approved:
 
 .. code-block:: bash
 
    $ git checkout main
    $ git pull
    $ release
-   $ git checkout -b start-X.Y.Z
-   $ postrelease
-
-The Mozilla remote-settings CI will immediately deploy the
-newly-tagged version to remote-settings stage and run the QA tests
-against it. Results are reported in the Mozilla ``#storage`` channel.
-
-Draft a release on Github:
-https://github.com/mozilla-services/kinto-dist/releases . For release
-notes, just use the CHANGELOG entry for the release, but change all
-the ReST-style section headings to Markdown-style ``##`` headings.
-
-Then:
 
-The "Back to development" commit cannot be pushed to main because we don't allow pushes to main.
-
-You can just throw away the commit (``git reset --hard HEAD^``) but
-the next person to touch the changelog will have to introduce a new
-heading for the next version. Another option is to push the commit and
-have it be reviewed:
+- At this point the package is published on Pypi. Now prepare the next version and push the tag to the repo with:
```

Comment:
> At this point the package is published on Pypi.

Is this still something we want to do?

Reply:
I know that we don't want to do this anymore, but I felt obliged to reflect the current state of the repo.

Reply:
Makes sense. Can we create an issue to track the work for removing this step? Or do we already have one?

Reply:
I opened https://github.com/mozilla-services/kinto-dist/issues/1960 :ok_hand: